apps/src/sites/hourofcode.com/pages/public/index.js

```diff
@@ -170,6 +170,8 @@ function showCensusForm(data) {
   $('#census-header').show();
   $('#thanks-header').show();
   $('#census-form').show();
+  // Jump up to the top of the form
+  window.location.hash = '#thanks-header';
```

Comment:
We could also do a smooth scroll which often looks nice.

https://github.com/code-dot-org/code-dot-org/blob/7c0d72cd7a5f1cad27dedd03b9244c1c697d2942/shared/js/helpers.js#L4

Reply:
I didn't know about that. I tried it out and I actually like the original approach better. The scroll doesn't really make sense here since we are swapping out one form for the other rather than moving to another spot in the same form.